runtime/include/j9cfg.h.ftl

```diff
@@ -42,7 +43,13 @@ extern "C" {
 #undef EsVersionMinor
 #define EsVersionMinor ${uma.buildinfo.version.minor}0
 
-#define EsVersionString "${uma.buildinfo.version.major}.${uma.buildinfo.version.minor}"
+#undef EsVersionString
+#ifdef VENDOR_VERSION_STRING
+#define EsVersionString VENDOR_VERSION_STRING
+#else
+#define EsVersionString "0.8.0-internal.j9cfg"
```

Comment:
I'm not sure I understand the `0.8.0-internal.j9cfg` value.  Does that get replaced by some other process?

Reply:
Sorry, my intermediate change was pushed to this branch, should be updated soon.

Reply:
Pls have another look.